timescaledb/tutorials/analyze-intraday-stocks/fetch-and-ingest.md

```diff
@@ -169,8 +225,7 @@ conn = psycopg2.connect(database=config.DB_NAME,
                         port=config.DB_PORT)
 
 # column names in the database (pgcopy needs it as a parameter)
-COLUMNS = ('time', 'symbol', 'price_open', 'price_close',
-           'price_low', 'price_high', 'trading_volume')
+COLUMNS = ('time', 'symbol', 'price_open', 'price_close', 'price_low', 'price_high', 'trading_volume')
```

Comment:
Furthermore, even after these fixes, the code will still fail with a ValueError exception because the columns are listed in the wrong order. Currently the code shows



Based on the comments in the issue https://github.com/timescale/docs/issues/1193:
COLUMNS = ('time', 'symbol', 'price_open', 'price_close',
           'price_low', 'price_high', 'trading_volume')
but the previous code snippet (i.e. the fetch_stock_data function) adds the symbol column last, so when pgcopy tries to parse the rows, it gets a float when expecting a string (the 'symbol'). Hence, the correct order is:

COLUMNS = ('time', 'price_open', 'price_close',
           'price_low', 'price_high', 'trading_volume', 'symbol')

Reply:
The `COLUMNS` needs to match with what's inside the database so it shouldn't be changed